lib/http-proxy/common.js

```diff
@@ -46,6 +46,10 @@ common.setupOutgoing = function(outgoing, options, req, forward) {
     extend(outgoing.headers, options.headers);
   }
 
+  if(options.auth){
```

Comment:
Please follow the code style in this document (adding spaces around the parentheses)